src/main.js

```diff
@@ -25,7 +25,7 @@ Vue.filter('stat', function (value) {
 })
 Vue.filter('aggressive', function (value) {
 	if (value == undefined) return 0;
-	if (value >= 1000) {
+	if (value >= 1000 && !rootGetters["settings/showFullValues"]) {
```

Comment:
`rootGetters` won't be defined here. I think you can do `store.rootGetters` though (don't quote me on this)